packages/app-mobile/utils/fs-driver/fs-driver-rn.ts

```diff
@@ -196,7 +221,9 @@ export default class FsDriverRN extends FsDriverBase {
 	}
 
 	public readFile(path: string, encoding = 'utf8') {
+		encoding = normalizeEncoding(encoding);
 		if (encoding === 'Buffer') throw new Error('Raw buffer output not supported for FsDriverRN.readFile');
```

Comment:
```suggestion
		if (encoding === 'buffer') throw new Error('Raw buffer output not supported for FsDriverRN.readFile');
```

1. Encodings are lowercase after normalization
2. fs-driver-node (I think) compares with lowercase `buffer`